keras_cv/metrics/coco/recall.py

```diff
@@ -12,11 +12,173 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import tensorflow as tf
+import tensorflow.keras as keras
+import tensorflow.keras.initializers as initializers
 
-from keras_cv.metrics.coco.base import COCOBase
+from keras_cv.metrics.coco import iou as iou_lib
+from keras_cv.metrics.coco import utils
+from keras_cv.utils import bbox
 
 
-class COCORecall(COCOBase):
+class COCORecall(keras.metrics.Metric):
+    """COCORecall computes the COCO recall metric.
+
+    Args:
+        iou_thresholds: defaults to [0.5:0.05:0.95].
+        category_ids: no default, users must provide.
```

Comment:
Could u provide more details about the category_ids? Its unclear to me how it is going to be used.